static/app/views/organizationContext.tsx

```diff
@@ -343,11 +350,13 @@ class OrganizationContext extends React.Component<Props, State> {
   renderBody() {
     return (
       <DocumentTitle title={this.getTitle()}>
-        <div className="app">
-          {this.state.hooks}
-          {this.renderSidebar()}
-          {this.props.children}
-        </div>
+        <OrganizationContext.Provider value={this.state.organization}>
```

Comment:
Nice! I don't think there are any remaining consumers of the organization legacy context. This could be a good point to remove that as well.

Reply:
@markstory isn't https://github.com/getsentry/sentry/blob/master/static/app/utils/withOrganization.tsx#L19 still using it?

Reply:
It could, I'm not looking to tread too much new ground in this PR though.